packages/PyTrilinos2/cmake/PyTrilinos2MakeTest.cmake

```diff
@@ -56,6 +56,6 @@ MACRO(PyTrilinos2_MAKE_MPI_TEST TEST_NAME)
     )
 
   tribits_set_tests_properties(PyTrilinos2_${TEST_NAME}_MPI_4
-    PROPERTIES ENVIRONMENT "${PYTHON_LIBRARY_PREFIX}:${PyTrilinos2_PYTHONPATH}")
+    PROPERTIES ENVIRONMENT "${PyTrilinos2_PYTHONPATH}")
 
```

Comment:
Can we merge this into the `TRIBITS_ADD_TEST` above by adding a `ENVIRONMENT "${PyTrilinos2_PYTHONPATH}"` line to it? cause that should still work for tests that do not run on 4 MPI ranks.